caffe2/serialize/versions.h

```diff
@@ -64,7 +64,7 @@ constexpr uint64_t kMaxSupportedFileFormatVersion = 0x6L;
 //     1. aten::div is changed at version 4
 //     2. aten::full is changed at version 5
 //     3. torch.package uses version 6
-constexpr uint64_t kProducedFileFormatVersion = 0x7L;
+constexpr uint64_t kProducedFileFormatVersion = 0x8L;
```

Comment:
Should the comments here and below be updated?

Reply:
Yes will do once I make all the tests pass